app/src/main/java/com/hisham/ctintegrationsample/BaseFragment.kt

```diff
@@ -1,11 +1,17 @@
 package com.hisham.ctintegrationsample
 
 import androidx.annotation.DrawableRes
+import androidx.appcompat.widget.Toolbar
+import androidx.core.content.ContextCompat
 import androidx.fragment.app.Fragment
 
 open class BaseFragment : Fragment() {
 
     fun toolbarIcon(@DrawableRes drawableResId: Int) {
-        requireActivity().actionBar?.setLogo(drawableResId)
+        requireActivity().findViewById<Toolbar>(R.id.toolbar).logo = if (drawableResId != 0) {
+            ContextCompat.getDrawable(requireContext(), drawableResId)
+        } else {
+            null
+        }
```

Comment:
Seems like we downgraded some dependencies this no longer worked.

Reply:
Can you revert it to the old code?